rpcs3/Emu/System.cpp

```diff
@@ -1221,6 +1218,12 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 				}
 
 				sys_log.notice("Elf path: %s", argv[0]);
+
+				// Load optional arguments from config
+				for (const std::string& arg : g_cfg.core.cli_args.get_vec())
```

Comment:
Maybe this is better?
```suggestion
				const auto& args = g_cfg.core.cli_args.get_vec();
				argv.insert(argv.end(), args.begin(), args.end());
```